RoomLocator/RoomLocator.Data/Services/FeedbackService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using Microsoft.EntityFrameworkCore;
+using RoomLocator.Data.Config;
+using RoomLocator.Domain;
+using RoomLocator.Domain.InputModels;
+using RoomLocator.Domain.Models;
+using RoomLocator.Domain.ViewModels;
+using Shared;
+
+namespace RoomLocator.Data.Services
+{
+    /// <summary>
+    ///     <author>Hamed Kadkhodaie, s083485</author>
+    /// </summary>
+    /// 
+    public class FeedbackService : BaseService
+    {
+        public FeedbackService(RoomLocatorContext context, IMapper mapper) : base(context, mapper) { }
+
+        public async Task<FeedbackViewModel> GetByDownvote(bool downvote)
```

Comment:
Man vil nok ikke være interesseret i at hente downvotes og upvotes hver for sig. Det bør være en liste med alle votes. Hvis man vil have en form for filtrering, så kunne det være på dato eller noget, men hold det simpelt til at starte med

Reply:
👍